### ISSUE TITLE
checkpoint save incorrectly updates best_loss instead of best_iou

### ISSUE THREAD

**Issue by ACC-Tony** [ChenYichen9527/EV-UAV#29]
[https://github.com/ChenYichen9527/EV-UAV/blob/main/train.py](url)
in line 113 of train.py 
`best_loss = loss.item()` 
shouldn't this line of code being `best_iou = iou.item()`, since it detects if the iou better than previous best value, and updates the best_iou value?

original code block:

``` python
        with torch.no_grad():
            if epoch>=40:
                for sample, ev in enumerate(val_dataloader):
                    x = ev['voxel_ev']
                    label = ev['seg_label'].float().cuda()
                    p2v_map = ev['p2v_map'].long().cuda()
                    ev_locs = ev['locs'].float().requires_grad_()
                    idx = ev['idx_label']
                    ts = ev_locs[:, 3]

                    preds, voxel = net(x)
                    preds = preds[p2v_map].squeeze().cpu()

                    evaluter.matches[str(sample)] = {}
                    evaluter.matches[str(sample)]['seg_pred'] = preds
                    evaluter.matches[str(sample)]['seg_gt'] = label
                iou = evaluter.evaluate_semantic_segmantation_miou()

                if iou.item() > best_iou:
                    torch.save(net.state_dict(), cfg.model_save_root + '/best_iou_seed{}.pt'.format(seed))
                    best_loss = loss.item()
```

**Comment by ChenYichen9527** [comment]
Thanks, we've fixed the issue in the code.
